packages/vue-components/src/Navbar.vue

```diff
@@ -36,6 +36,15 @@ import normalizeUrl from './utils/urls';
 import SiteNavButton from './SiteNavButton.vue';
 import PageNavButton from './PageNavButton.vue';
 
+function isEqualExceptLast(hParts, uParts) {
```

Comment:
Just a minor suggestion: would it be better to place this in `methods`?

Reply:
👍 shifted back in for consistency